modules/http/tests/http_split_response.cc

```diff
@@ -22,11 +22,7 @@
 #include <boost/test/unit_test.hpp>
 
 #include "../http.h"
-
-
-extern const gchar *last_log;
-extern const gchar *last_class;
-extern int last_level;
+#include "testutil.h"
```

Comment:
Seems that you forgot to commit `testutil.h` that contains the definition of `testErrorCases()`.